client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -126,18 +126,22 @@ class ImageEditorCanvas extends Component {
 
 		this.drawImage();
 		this.updateCanvasPosition();
-		this.onWindowResize = throttle( this.updateCanvasPosition, 200 );
+
 		if ( typeof window !== 'undefined' ) {
 			window.addEventListener( 'resize', this.onWindowResize );
 		}
 
 		this.props.setImageEditorImageHasLoaded( this.image.width, this.image.height );
 	}
 
+	onWindowResize() {
+		this.requestAnimationFrameId = window.requestAnimationFrame( this.updateCanvasPosition );
```

Comment:
Thanks for the tip! I was going to implement something similar to this:
https://developer.mozilla.org/en-US/docs/Web/Events/resize#requestAnimationFrame

But would still to expose the frameId ref. 

Hypothetical: is client/libs the best place for new utils?